packages/integration-test-app/windows/integrationtest/integrationtest.vcxproj

```diff
@@ -13,16 +13,15 @@
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
-    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.19041.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
     <AppxPackageSigningEnabled>false</AppxPackageSigningEnabled>
   </PropertyGroup>
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="ReactNativeWindowsProps">
     <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'node_modules\react-native-windows\package.json'))\node_modules\react-native-windows\</ReactNativeWindowsDir>
     <ConsumeCSharpModules Condition="'$(ConsumeCSharpModules)' == ''">true</ConsumeCSharpModules>
     <BundleEntryFile Condition="'$(BundleEntryFile)' == ''">index.tsx</BundleEntryFile>
   </PropertyGroup>
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.WindowsSdk.Default.props" />
```

Comment:
`Microsoft.Cpp.Default.props` will override `WindowsTargetPlatformVersion` if not set, so we must set it fairly early. This is why it is included separately from the other shared props.

Reply:
What is "External"?

Is it a set of props files meant for consuming projects as opposed to core RNW building?

Reply:
@jonthysell came up with the original naming, but they are used to define properties in our npm package that user apps/Libraries include, instead of having the property sheets in the user app. This means that props automatically change in the user app if they update their npm package.

They are included by projects external to MSRN, that include MSRN.